packages/zoe/src/contractSupport/prepare-ownable.js

```diff
@@ -38,7 +38,7 @@ const TransferProposalShape = M.splitRecord({
  *   The method names of the underlying exo class that should be represented
  *   by transparently-forwarding methods of the wrapping ownable object.
  * @param {OwnableOptions} [options]
- * @returns {<U>(underlying: U) => Pick<U, MN[number]> & {makeTransferInvitation: () => Invitation<U>}}
+ * @returns {<U>(underlying: U) => Partial<U> & {makeTransferInvitation: () => Invitation<U>}}
```

Comment:
`Pick<U, …>` → `Partial<U>` looks like a regression of developer experience.

Reply:
With that reversion, `yarn lint` gives me

```
test/unitTests/contracts/ownable-counter.test.js:40:39 - error TS2339: Property 'toBeAttenuated' does not exist on type 'EMethods<Required<object & Pick<Guarded<{ incr(): bigint; getInvitationCustomDetails(): { count: bigint; }; toBeAttenuated(): void; }>, "incr" | "getInvitationCustomDetails"> & { ...; }>>'.

40   await t.throwsAsync(E(firstCounter).toBeAttenuated(), {
                                         ~~~~~~~~~~~~~~
```

so backing out of that too. No doubt that type error is why I made this change in the first place.

Reply:
Resolving.